source/sn_coap_protocol.c

```diff
@@ -2509,3 +2515,32 @@ static sn_coap_hdr_s *sn_coap_protocol_copy_header(struct coap_s *handle, sn_coa
     return destination_header_ptr;
 }
 #endif
+
+#if SN_COAP_DUPLICATION_MAX_MSGS_COUNT
+static bool sn_coap_protocol_update_duplicate_package_data( struct coap_s *handle,
+                                                        sn_nsdl_addr_s *dst_addr_ptr,
+                                                        sn_coap_hdr_s *coap_msg_ptr,
```

Comment:
These should be "const sn_.. *" parameters.

Reply:
Fixed.